.github/workflows/check-dist.yml

```
@@ -0,0 +1,43 @@
+# `dist/index.js` is a special file in Actions.
+# When you reference an action with `uses:` in a workflow,
+# `index.js` is the code that will run.
+# For our project, we generate this file through a build process
+# from other source files.
+# We need to make sure the checked-in `index.js` actually matches what we expect it to be.
+name: Check dist/
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  check-dist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - run: npm install
+      - run: mv dist/index.js /tmp
+      - run: npm run release
+      - run: git diff --ignore-all-space dist/index.js /tmp/index.js
```

```suggestion
      - name: Install dependencies
        run: npm install
      - name: Move checked in index.js file to tmp directory
        run: mv dist/index.js /tmp
      - name: Create new index.js file
        run: npm run release
      - name: Compare expected vs actual index.js file
        run: git diff --ignore-all-space dist/index.js /tmp/index.js
```

Nit but I like to have names for all the steps so the log view looks better